.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+name: Release
+
+on:
```

Comment:
here the trigger could be `on: release`
see https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages#publishing-packages-to-the-npm-registry

Reply:
`on: release` implies we create releases on GitHub, which I'm not sure we want to do.
Solely relying on tag and branch seems simpler and does not require interacting with GitHub.

Reply:
github helps with tagging, I have been used Github Releases for other projects, and it doesn't add too much complexity.

Reply:
I've updated the release condition to be `on: release: types: [published]`